jaxrs/src/main/resources/kafka/consumer.properties

```diff
@@ -0,0 +1,13 @@
+# A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. This list only impacts
+# the initial hosts used to discover the full set of servers. This list should be in the form host1:port1,host2:port2,...
+# If you run Kafka locally set to localhost:9092, if you run Kafka on Docker, follow the instructions in the project and set to YOUR_IP:9092 or again localhost:9092 depending on the OS
+bootstrap.servers=localhost:9092
+# A unique string that identifies the consumer group this consumer belongs to. This property is required if the consumer
+# uses either the group management functionality by using subscribe(topic) or the Kafka-based offset management strategy.
+# Using it for a consumers that reads from "installationMetrics" topic allows only unread messages from other consumers
+# to be consumed.
+group.id=installationMetricsGroup
+# Deserializer class for key
+key.deserializer=org.apache.kafka.common.serialization.StringDeserializer
+# Deserializer class for value
+value.deserializer=org.apache.kafka.common.serialization.StringDeserializer
```

Comment:
We should probably add `auto.offset.reset=earliest` because the producer sends to the topic before the consumer is started